custom/icds_reports/utils/aggregation_helpers/child_health_monthly.py

```diff
@@ -54,14 +54,14 @@ def person_case_ucr_tablename(self):
 
     @property
     def tablename(self):
-        return "{}_{}".format(self.base_tablename, self.month.strftime("%Y-%m-%d"))
```

Comment:
@calellowitz where was this table getting created before?

Reply:
In the tasks file. There was a sql file that did it.

Reply:
https://github.com/dimagi/commcare-hq/pull/23790/files#diff-b8c5a5e308a203bcaee8ed83e6837edfL471